examples/loadtest/cli/cli_e2e_test.go

```diff
@@ -0,0 +1,45 @@
+//go:build e2e
+
+package main
+
+import (
+	"syscall"
+	"testing"
+	"time"
+
+	"go.uber.org/goleak"
+
+	"github.com/hatchet-dev/hatchet/internal/testutils"
+)
+
+func TestCLI(t *testing.T) {
+	testutils.Prepare(t)
+	defer func() {
+		if err := syscall.Kill(syscall.Getpid(), syscall.SIGINT); err != nil {
+			t.Fatalf("syscall.Kill() error = %v", err)
+		}
+
+		time.Sleep(2 * time.Second)
+
+		goleak.VerifyNone(
+			t,
+			goleak.IgnoreTopFunction("go.opencensus.io/stats/view.(*worker).start"),
+			goleak.IgnoreTopFunction("github.com/hatchet-dev/hatchet/pkg/cmdutils.InterruptChan.func1()"),
+			goleak.IgnoreTopFunction("google.golang.org/grpc/internal/grpcsync.(*CallbackSerializer).run"),
+			goleak.IgnoreTopFunction("internal/poll.runtime_pollWait"),
+			goleak.IgnoreTopFunction("google.golang.org/grpc/internal/transport.(*controlBuffer).get"),
+		)
+	}()
+
+	//goleak.IgnoreCurrent()
```

Comment:
rm dead code

Reply:
This is from my branch, we have a bit of a messed up tree now... @abelanger5 I will merge my changes into `main`, potentially you could just drop those commits from this branch to also prevent any future conflicts.